Tools/WebKitTestRunner/cocoa/UIScriptControllerCocoa.mm

```diff
@@ -353,4 +355,28 @@ - (void)paste:(id)sender;
     }];
 }
 
+void UIScriptControllerCocoa::adjustVisibilityForFrontmostTarget(int x, int y, JSValueRef callback)
+{
+    unsigned callbackID = m_context->prepareForAsyncTask(callback, CallbackTypeNonPersistent);
+    auto request = adoptNS([_WKTargetedElementRequest new]);
+    [request setPoint:CGPointMake(x, y)];
+    [webView() _requestTargetedElementInfo:request.get() completionHandler:[callbackID, this](NSArray<_WKTargetedElementInfo *> *elements) {
+        if (!elements.count) {
+            m_context->asyncTaskComplete(callbackID);
+            return;
+        }
+
+        RetainPtr<_WKTargetedElementInfo> frontTarget = elements.firstObject;
+        [webView() _adjustVisibilityForTargetedElements:@[frontTarget.get()] completionHandler:[callbackID, frontTarget, this] (BOOL success) {
```

Comment:
Nit:
```suggestion
        [webView() _adjustVisibilityForTargetedElements:@[ frontTarget.get() ] completionHandler:[callbackID, frontTarget, this] (BOOL success) {
```

Reply:
Ah, I had that originally, but the style checker (erroneously) complained about extra spaces around the "capture group" 😓. I'll add the spaces back in (and at some point, we should probably teach the style checker about `@[]`…)

Thanks for the review!